### DISCUSSION TITLE
plugin controller

### DISCUSSION THREAD

**Discussion by frascal** [shanteacontrols/OpenDeck#199] [Q&A]
Good morning,
I would like to build a plugin controller based on the ssl channel strip model.
I ask myself the question: can opendeck act only on the selected plugin?
Because if I work on a project with 1 ssl strip per channel, I want to change the parameters of a single plugin and not of all the plugins.
My music software is reaper.
Thanks for your advice.

**Comment by paradajz** [comment]
I have no idea what are you asking here. OpenDeck is a standard MIDI device as far as your PC or applications are concerned. Check the documentation of the software you wish to use it for.

**Reply by frascal** [reply]
Sorry if I expressed myself badly.
I use google translate to communicate my message to you.
I mean:
If I use reaper for example and I want to control a plugin (which I have selected) will it only act on this instance?
